src/windows/SliderWindows/SliderView.cpp

```diff
@@ -237,4 +235,13 @@ namespace winrt::SliderWindows::implementation {
                 });
         }
     }
+
+    const double SliderView::CalculateStepFrequencyPercentageValue(const double& stepPropertyValue) const noexcept {
+        if (stepPropertyValue != 0) {
+            return stepPropertyValue;
+        }
+        else {
+            return (m_maxValue - m_minValue) / 100.f;
```

Comment:
don't need '.f' since m_maxvalue is already a floating-point type

Reply:
Well... I've implemented it when there was still no your parallel fix delivered, so both `m_maxValue` and `m_minValue` were integers, which resulted in warning without `.f` modifier.
You've got the point here - this is a merge mistake done by me, because when I was merging our solutions I completely forgot that you have changed the types of min and max values (though I intentionally left that part of your delivery as very usefull change).